### PR TITLE
master-lps-105582

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-108603
https://issues.liferay.com/browse/LPS-108606

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: modules/apps/depot/depot-test/src/testFunctional/tests/DepotContentSets.testcase [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+		MenuItem.viewPresent(menuItem = "Manual Selection");
+
+		MenuItem.viewPresent(menuItem = "Dynamic Selection");
+
+		Refresh();
+
+		Click(locator1 = "Message#EMPTY_INFO_TAGLIB");
+
+		Click.waitForMenuToggleJSClick(locator1 = "Dropdown#EMPTY_MESSAGE_DROPDOWN");
```
Comment (austinchiang) [inline modules/apps/depot/depot-test/src/testFunctional/tests/DepotContentSets.testcase]: I think this is not necessary if we are just asserting that the elements below are there. If that is the case we can create a test improvement for this; I'll push it in for now.

**Comment by austinchiang** [comment]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/84652

**Pull request closed without merging**